packages/common/driver-definitions/src/storage.ts

```diff
@@ -298,6 +299,16 @@ export interface IDocumentDeltaConnection
 	// TODO: Use something other than `any`.
 	// eslint-disable-next-line @typescript-eslint/no-explicit-any
 	submitSignal(content: any, targetClientId?: string): void;
+
+	/**
+	 * Tells whether the connection supports connection updates events on the connection object
+	 */
+	supportDeltaConnectionUpdates?: boolean;
```

Comment:
Is this property required mostly for staging, to avoid exception when registering "deltaConnectionUpdated" exception with old loaders? if so, should we remove it in the future (open a task to track it)?

Reply:
Will open a task. Yes it was added to avoid the exception for the event.

Reply:
Removed it since no longer needed after emitting event on service.